package.json

```diff
@@ -10,6 +10,7 @@
     "gavel": "bin/gavel"
   },
   "scripts": {
+    "lint": "eslint lib/**/*.js",
```

Comment:
not linting tests?

Reply:
Good point. Added under 837df08.
I want to also add some linting in a watch mode, since the library doesn't have a compilation step, I'd like to prevent finding out about linting errors in CI. Needs to fail fast on local, while developing.

Reply:
👍